docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,115 +1,150 @@
-
-=== Use a basic preset
+$ ls
 
-Presets are ready-to-use contracts that you can deploy right away.
-They also serve as examples of how to use library modules.
-xref:extensibility.adoc#presets[Read more about presets].
-
-[,cairo]
-----
-// contracts/MyToken.cairo
-
-%lang starknet
-
-from openzeppelin.token.erc20.presets.ERC20 import (
-    constructor,
-    name,
-    symbol,
-    totalSupply,
-    decimals,
-    balanceOf,
-    allowance,
-    transfer,
-    transferFrom,
-    approve,
-    increaseAllowance,
-    decreaseAllowance
-)
+Scarb.toml src
 ----
 
-Compile and deploy it right away:
+=== Install the library
 
-[,bash]
-----
-nile compile
+Install the library by declaring it as a dependency in the project's `Scarb.toml` file:
 
-nile deploy MyToken <name> <symbol> <decimals> <initial_supply> <recipient> --alias my_token
+[,text]
+----
+[dependencies]
+openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.7.0" }
 ----
 
-NOTE: `<initial_supply>` is expected to be two integers i.e.
-`1` `0`.
-See xref:utilities.adoc#uint256[uint256] for more information.
+WARNING: Make sure the tag matches the target release.
 
-=== Write a custom contract using library modules
+== Basic usage
 
-xref:extensibility.adoc#libraries[Read more about libraries].
+This is how it looks to build an account contract using the xref:accounts.adoc[account module].
+Copy the code into `src/lib.cairo`.
 
-[,cairo]
+[,javascript]
+----
+#[starknet::contract]
+mod MyAccount {
+    use openzeppelin::account::Account;
+    use openzeppelin::account::account::PublicKeyTrait;
+    use openzeppelin::account::interface;
+    use openzeppelin::introspection::interface::ISRC5;
+    use openzeppelin::introspection::src5::SRC5;
+    use starknet::account::Call;
+
+    // Storage members used by this contract are defined in each imported
+    // module whose `unsafe_state` is used. This design will be improved
+    // with the addition of components in the future.
+    #[storage]
+    struct Storage {}
+
+    #[constructor]
+    fn constructor(ref self: ContractState, public_key: felt252) {
+        let mut unsafe_state = _unsafe_state();
+        Account::InternalImpl::initializer(ref unsafe_state, public_key);
+    }
+
+    #[external(v0)]
+    impl SRC6Impl of interface::ISRC6<ContractState> {
+        fn __execute__(self: @ContractState, mut calls: Array<Call>) -> Array<Span<felt252>> {
+            Account::SRC6Impl::__execute__(@_unsafe_state(), calls)
+        }
+
+        fn __validate__(self: @ContractState, mut calls: Array<Call>) -> felt252 {
+            Account::SRC6Impl::__validate__(@_unsafe_state(), calls)
+        }
+
+        fn is_valid_signature(
+            self: @ContractState, hash: felt252, signature: Array<felt252>
+        ) -> felt252 {
+            Account::SRC6Impl::is_valid_signature(@_unsafe_state(), hash, signature)
+        }
+    }
+
+    #[external(v0)]
+    impl SRC5Impl of ISRC5<ContractState> {
```

Comment:
I think we can use the `ISRC5` imported in `Account` e.g. `Account::ISRC5`. This will save us from importing the interface. I lean toward not doing this because it's better to import the interface from where it's defined. Just throwing it out there in case there's differing opinions

Reply:
mmm, good point. I think I like the most importing the interface from the main introspection module, but no strong defense.

Reply:
+1 to direct import, this is to showcase how to use the library and using that indirection might be confusing and hinder feature discoverability